apps/framework-cli/src/framework/flows/loader.rs

```diff
@@ -51,27 +52,77 @@ async fn get_all_flows(
+        if source.metadata()?.is_file() {
+            if !source.file_name().to_str().unwrap().ends_with("sql") {
+                let potential_flow_file_name = &source
+                    .path()
+                    .with_extension("")
+                    .file_name()
+                    .unwrap()
+                    .to_string_lossy()
+                    .to_string();
+
+                match migration_regex.captures(potential_flow_file_name) {
+                    None => {
+                        let split: Vec<&str> = potential_flow_file_name.split("__").collect();
+                        if split.len() == 2 {
+                            let source_data_model_name = split[0];
+                            let target_data_model_name = split[1];
+
+                            let source_data_model = if let Some(source_data_model) =
+                                data_models.get(&source_data_model_name, current_version)
+                            {
+                                source_data_model
+                            } else {
+                                warn!(
+                                    "Data model {} not found in the data model set",
+                                    source_data_model_name
+                                );
+                                continue;
+                            };
+
+                            let target_data_model = if let Some(target_data_model) =
+                                data_models.get(&target_data_model_name, current_version)
+                            {
+                                target_data_model
+                            } else {
+                                warn!(
+                                    "Data model {} not found in the data model set",
+                                    target_data_model_name
+                                );
+                                continue;
+                            };
+
+                            let flow = Flow {
+                                name: potential_flow_file_name.clone(),
+                                source_data_model: source_data_model.clone(),
+                                target_data_model: target_data_model.clone(),
+                                executable: source.path(),
+                                version: current_version.to_string(),
+                            };
+                            flows.push(flow);
+                        } else {
+                            debug!(
+                                "Fragments of file {:?} does not match the convention",
+                                source.path()
+                            );
+                        }
+                    }
+                    Some(caps) => {
+                        let flow = build_migration_flow(
+                            &source.file_name().to_string_lossy(),
+                            data_models,
+                            current_version,
```

Comment:
Not sure what version should mean in a migration flow.

Reply:
That's a good question - should we make a different type of flow that not versioned for migrations?

Reply:
I don't think I have a good answer